http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/RuntimeServiceInterp.scala

```diff
@@ -92,21 +92,30 @@ class RuntimeServiceInterp[F[_]: Parallel](config: RuntimeServiceConfig,
                 ) as None
             }
             clusterImages <- getRuntimeImages(petToken, context.now, req.toolDockerImage, req.welderDockerImage)
-            gceRuntimeDefaults = config.gceConfig.runtimeConfigDefaults
+            diskOpt <- req.diskConfig.traverse(x =>
+              processDiskConfigRequest(x,
+                                       googleProject,
+                                       req.runtimeConfig.map(_.cloudService).getOrElse(CloudService.GCE),
+                                       userInfo)
+            )
+            // For GCE, make diskConfig.size take precedence over runtimeConfig.diskSize
+            gceRuntimeDefaults = diskOpt match {
```

Comment:
do we still need the old config if we now have diskConfig?

Reply:
We still need `runtimeConfig`, just `runtimeConfig.diskSize` is redundant now. We can remove it eventually but we need to keep it for now because disks aren't _actually_ implemented yet in back Leo :)